superset/examples/countries.md

```diff
@@ -212,10 +212,10 @@ SP.DYN.LE00.IN | Life expectancy at birth, total (years)
 SP.DYN.LE00.MA.IN | Life expectancy at birth, male (years)
 SP.DYN.SMAM.FE | Mean age at first marriage, female
 SP.DYN.SMAM.MA | Mean age at first marriage, male
-SP.DYN.TFRT.IN | Fertility rate, total (births per woman)
+SP.DYN.TFRT.IN | Fertility rate, total (births per person)
```

Comment:
Actually in this case I think "woman" is the correct framing since it's focused on fertility rate!